utils/lit/lit/formats/taef.py

```diff
@@ -38,6 +38,10 @@ def executeCommandForTaef(command, cwd=None, env=None):
 
 class TaefTest(TestFormat):
     def __init__(self, te_path, test_dll, test_path, select_filter, extra_params):
+        # Try to find te in %PATH% if not exist.
+        if not os.path.exists(te_path):
+            te_path = lit.util.which('TE')
+
         self.te = te_path
```

Comment:
Should we raise an exception here if the te_path does not exist?

Reply:
It will throw exception later when use self.te and print error like this.
 error: unable to discover taef tests in 'D:/agent/_work/2/b/DXC\\Debug\\bin\\ClangHLSLTests.dll', using TAEF_EXECUTABLE-NOTFOUND